agents/email_agent.py

```
@@ -0,0 +1,333 @@
+                    filename = part.get_filename()
+                    if filename:
+                        email_data["attachments"].append(filename)
+        else:
+            email_data["body"] = msg.get_payload(decode=True).decode('utf-8', errors='ignore')
+        
+        return email_data
+    
+    async def categorize_email(self, email_data: Dict[str, Any]) -> str:
+        """Categorize an email based on its content (simple rule-based approach)"""
+        subject = email_data["subject"].lower()
+        body = email_data["body"].lower()
```

Ruff (0.12.2)

Local variable `body` is assigned to but never used

Remove assignment to unused variable `body`

(F841)